### PR TITLE
docs: Chinese documentation optimisation

### PR DESCRIPTION
**Pull request by WingLim**
Still, there are some format style issue, some words issue and more.
I am optimising the document to let Chinese reader have better experience.
This may take some time to do it. And need @linbuxiao your help here.

## Format
- [x] No space before or after the Chinese punctuation
- [x] Add space for link
- [x] Use Chinese brackets only for the content

## Translation
- [x] Better and easier translation to understand

### CODE REVIEW

**Review by linbuxiao — APPROVED** [review]
The details are the hardest part. Through your efforts, new specifications have been established. It was a very difficult task and admire you for accomplishing this.

File: site/docs/zh/guide/api.md [hunk 1]
```diff
@@ -8,57 +8,57 @@ next: ./filter-queries.md
...
-它的可以被看作这样：
+请求的步骤可以被可视化为这样：
 
 ```asciiart:no-line-numbers
 ( ( ( Telegram ) MTProto API ) Bot HTTP API ) <-- bot 在此连接
 ```
 
-换句话说：当你的 bot 发送消息时，它将以 HTTP 请求的形式发送到一个 _Bot API服务器_（由 Telegram 团队托管，或者[由你自己托管](https://core.telegram.org/bots/api#using-a-local-bot-api-server)）。
-这个服务器将把请求翻译成 Telegram 的本地协议，称为 MTProto ，并向 Telegram 的后端发送请求，后者负责将信息发送给用户。
+换句话说：当你的机器人发送消息时，它将以 HTTP 请求的形式发送到一个 _Bot API服务器_（由 Telegram 团队托管，或者 [由你自己托管](https://core.telegram.org/bots/api#using-a-local-bot-api-server)）。
```
Comment (WingLim) [inline site/docs/zh/guide/api.md:21]:
```suggestion
换句话说：当你的 bot 发送消息时，它将以 HTTP 请求的形式发送到一个 _Bot API服务器_（由 Telegram 团队托管，或者 [由你自己托管](https://core.telegram.org/bots/api#using-a-local-bot-api-server)）。
```

**Review by KnorpelSenf — APPROVED** [review]
Code examples look good to me. Also, despite not being able to read much, I can see that this PR is very consistent in its changes. Great work!

File: site/docs/zh/guide/context.md [hunk 2]
```diff
@@ -388,19 +389,19 @@ type MyContext = SomeFlavorA<Context>;
 
 ### 结合不同类型的上下文调味剂
 
-如果你有不同的[添加式调味剂](#添加式上下文调味剂)，你可以像这样配置它们。
+如果你有不同的[添加式调味剂](#添加式上下文调味剂)，你可以像这样配置它们：
```
Comment (WingLim) [inline site/docs/zh/guide/context.md:392]:
```suggestion
如果你有不同的 [添加式调味剂](#添加式上下文调味剂)，你可以像这样配置它们：
```

File: site/docs/zh/guide/games.md [hunk 3]
```diff
@@ -24,15 +24,15 @@ Telegram 游戏是一个非常有趣的功能，很好玩。
 ## 通过 bot 发送游戏
 
 我们可以通过 `replyWithGame` 方法在 grammY 发送游戏，该方法以你用 BotFather 创建的游戏的名称作为参数。
-或者，我们也可以使用 `api.sendGame` 方法( grammY 提供了所有官方的 [Bot API](https://core.telegram.org/bots/api)方法）。
+或者，我们也可以使用 `api.sendGame` 方法（grammY 提供了所有官方的 [Bot API](https://core.telegram.org/bots/api)方法）。
```
Comment (WingLim) [inline site/docs/zh/guide/games.md:27]:
```suggestion
或者，我们也可以使用 `api.sendGame` 方法（grammY 提供了所有官方的 [Bot API](https://core.telegram.org/bots/api) 方法）。
```

File: site/docs/zh/guide/getting-started.md [hunk 4]
```diff
@@ -5,30 +5,30 @@ next: ./basics.md
...
-> 如果你不知道这些东西是什么，请查看我们的[简介](./introduction.md)!
+> 本指南假定你已经安装了 [Node.js](https://nodejs.org)，并且 `npm` 应该是自带的。
+> 如果你不知道这些东西是什么，请查看我们的 [简介](./introduction.md)！
 
 创建一个新的 TypeScript 项目并安装 `grammy` 包。
 通过打开终端并键入以下内容来完成。
 
 ```bash
-# 创建一个新的目录，并进入该目录
+# 创建一个新的目录，并进入该目录.
```
Comment (WingLim) [inline site/docs/zh/guide/getting-started.md:19]:
```suggestion
# 创建一个新的目录，并进入该目录。
```

File: site/docs/zh/guide/introduction.md [hunk 5]
```diff
@@ -77,19 +77,19 @@ bot 通过 HTTP 请求与 Telegram 互动。
 
 这里有一些关于编程的趣事--这些事情对编程是必不可少的，但大家却很少解释，因为大多数开发者认为它们是不言而喻的。
 
-在下一节中，你将通过编写一个包含编程语言 [TypeScript](https://www.typescriptlang.org/docs/handbook/typescript-from-scratch.html) 源代码的文本文件来创建一个 bot 。
+在下一节中，你将通过编写一个包含编程语言 [TypeScript](https://www.typescriptlang.org/docs/handbook/typescript-from-scratch.html) 源代码的文本文件来创建一个 bot。
 grammY 文档不会教你如何编程，所以我们希望你能自学。
-不过，请记住：用 grammY 创建 Telegram Bot 是个学习编程的好方法。恭喜你找到了他。 :rocket:
+不过，请记住：用 grammY 创建 Telegram Bot 是个学习编程的好方法。恭喜你找到了他。:rocket:
```
Comment (WingLim) [inline site/docs/zh/guide/introduction.md:82]:
```suggestion
不过，请记住：用 grammY 创建 Telegram Bot 是个学习编程的好方法。恭喜你找到了他。 :rocket:
```

**Review by WingLim — COMMENTED** [review]
Final adjustments.

**Pull request merged**